tests/unit/type/stream.tcl

```diff
@@ -61,6 +61,10 @@ start_server {
     test {XADD can add entries into a stream that XRANGE can fetch} {
         r XADD mystream * item 1 value a
         r XADD mystream * item 2 value b
+
+        # coverage for objectComputeSize
+        assert_range [r memory usage mystream] 4700 5000 ;# Range can't be lower than 20bytes lower than actual stream and higher than 50 bytes; mystream size here is 4730
```

Comment:
i don't follow that comment.
you mention 50 bytes and then use a constant of 4700.
is 4730 just what you measured? and then added some tolerance around that value?

if a nearly empty stream has such a huge overhead, and should obviously not be optimized for small streams, maybe we should test a larger one...
on the other hand, stream memory usage estimation is pretty inaccurate, but still, i suppose that when it'll be large the test will be less awkward.
p.s. a stream has a ton of metadata in it, so if we care about coverage, let's create various types of streams, including ones with consumer groups.
i.e. we also want to test one without consumer groups, so that if we have a missing null check, the test can uncover that.

Reply:
Hi Oran,
updated the test with 10000 items and also added with group, could you please have a look.